challenges-webapp/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -10,7 +10,7 @@
     <portlet-class>org.exoplatform.commons.api.portlet.GenericDispatchedViewPortlet</portlet-class>
     <init-param>
     <name>portlet-view-dispatched-file-path</name>
-    <value>/html/challenges.html</value>
+    <value>/WEB-INF/jsp/challenges.jsp</value>
     </init-param>
     <expiration-cache>-1</expiration-cache>
     <cache-scope>PUBLIC</cache-scope>
```

Comment:
Please delete public cache of the resultant HTML to avoid making the achievements Tab visible to all users